app/services/ads.js

```diff
@@ -11,8 +11,11 @@ export default Service.extend({
   siteHeadOffset: 0,
   slotNames: null,
   noAdsQueryParam: null,
+  disableAdsInMobileApp: null,
```

Comment:
@hakubo says we need `computed()` here but it doesn't work throwing weird errors in console about maximum stack... any ideas?